schemas/dex/schema.json

```diff
@@ -246,6 +246,10 @@
           "description": "The amount of the output token, decimal normalized.",
           "type": "number"
         },
+        "spot_price_after_swap": {
```

Comment:
This is nitpicking, but wdyt of just `spot_price`, do we need to specify after swap?

Reply:
There's a different spot price before, during, and after the swap, so this is more clear.